### PR TITLE
Nearsighted Camera Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Using cameras while wearing glasses as a nearsighted character no longer reduces your vision radius.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
!bugfix

**Review by alsoandanswer — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

File: code/modules/modular_computers/file_system/program.dm [hunk 1]
```diff
@@ -236,6 +236,12 @@
 	else
 		return -1
 
+// Relays the call to nano module, if we have one
+/datum/computer_file/program/proc/grants_equipment_vision(var/mob/user)
+	if(NM)
+		return NM.grants_equipment_vision(user)
+	return ..()
```
Comment (FluffyGhoster) [inline code/modules/modular_computers/file_system/program.dm:243]: If you are declaring the proc here, there's no parent to call

File: code/game/atoms.dm [hunk 2]
```diff
@@ -86,6 +86,9 @@
 		return 0
 	return -1
 
+/atom/proc/grants_equipment_vision(var/mob/user)
```
Comment (FluffyGhoster) [inline code/game/atoms.dm:89]: This should have a dmdoc that explains what is the purpose of this function, and what it returns

**Comment by Geevies** [comment]
!review

**Pull request merged**